packages/engine/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [workspace]
 members = [".", "bin/*", "lib/*"]
-default-members = [".", "bin/*", "lib/*"]
+default-members = [".", "lib/*"]
```

Comment:
I know it functionally doesn't have a difference but I'm not sure we want to make this change. Conceptually we want to build the libraries and binaries by default. It's actually quite weird that the libraries force the binaries to be rebuilt, but it's what we've wanted for a while so I'm not complaining about that.

I just think that if someone looks at default-members they should see we're building all of them by default.

Reply:
Good point.